src/Type/JWT/UserbaseJWTUserFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace FlexAuth\Type\JWT;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+use UserBase\Client\Model\Account;
+use UserBase\Client\Model\AccountUser;
+use UserBase\Client\Model\User;
+
+class UserbaseJWTUserFactory implements JWTUserFactoryInterface
```

Comment:
Probably move to `/Type/UserbaseClient` folder is better. All related to Userbase will be in there.